usim/_primitives/timing.py

```diff
@@ -420,46 +420,53 @@ def __repr__(self):
         yield loop.time
 
 
-def each(
-        *, delay: float = None, interval: float = None
-) -> AsyncIterable[float]:
+def interval(value: float) -> AsyncIterable[float]:
```

Comment:
I recommend renaming the ``value`` to a name associated with interval (and delay). "delay by", "delay of" and "interval of" would fit. Not sure whether both ``delay`` and ``interval`` should use the same name or a different one - the first feels easier to use, the latter more robust when refactoring.

Reply:
I now removed the keyword itself.